core/domain/skill_services.py

```diff
@@ -514,3 +514,86 @@ def save_skill_summary(skill_summary):
     )
 
     skill_summary_model.put()
+
+
+def create_skill_mastery(user_id, skill_id, degree_of_mastery):
+    """Creates and stores skill mastery of a user.
+
+    Args:
+        user_id: str. The user id of the logged in user.
+        skill_id: str. The unique id of the skill.
+        degree_of_mastery: float. The degree of mastery of user in the skill.
+    """
+    skill_mastery_model = skill_models.SkillsMasteryModel(
```

Comment:
I think you can do:

    user_skill_mastery = UserSkillMastery(user_id, skill_id, degree_of_mastery)
    save_user_skill_mastery(user_skill_mastery)

I.e. follow a similar pattern as for Explorations. Do all updates to the domain object and have a generic function that takes a domain object and saves it to storage.

Reply:
Okay